great_expectations/rule_based_profiler/rule/rule.py

```diff
@@ -108,6 +152,27 @@ def expectation_configuration_builders(
             for expectation_configuration_builder in self._expectation_configuration_builders
         }
 
+    @property
+    def domain_builder(self) -> DomainBuilder:
+        return self._domain_builder
+
+    @property
+    def parameter_builders(self) -> Optional[List[ParameterBuilder]]:
+        parameter_builders_as_dict: Dict[
```

Comment:
I'm a firm believer that variable names should reflect the contents but not their type. Naming this dict and then annotating it as Dict feels redundant.

Can we think of a better name for the dictionary representation of the object? Perhaps `serialized_parameter_builders`?

Reply:
This same note extends to method names (I feel less strongly about those though)

Reply:
@cdkini I cleaned up the naming to not have this redundancy (in next commit).  Thanks!